lib/database_rewinder.rb

```diff
@@ -82,16 +82,28 @@ def all_table_names(connection)
       end
     end
 
+    def get_config_from(connection_name)
+      if Gem::Version.new(Rails.version) >= Gem::Version.new("6.1.0.alpha")
```

Comment:
@eileencodes Should we check `configs_for` change for Rails 6.0 or 6.1? I found the refactoring introduced in rails/rails#32274 which says `v6.0.0.beta1`.

Reply:
`configs_for` has been available since 6.0. The hash implementation will be removed in 6.2.